qdrant-landing/content/documentation/infrastructure/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Infrastructure
-weight: 21
-partition: build
+weight: 29
+partition: cloud
```

Comment:
I don't feel strongly about giving 3rd party tools their own section in the `Cloud` page.

Reply:
If it's cloud specific, like Terraform or Pulumi it should be under cloud. It's even not 100% 3rd party, since we develop the terraform provider against our cloud API.